src/components/appointmentsTable/tableRow.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import { TableRow, TableCell, Checkbox } from '@mui/material';
+import { bodyCell } from './style';
+import axiosInstance from '../../utils/apis/axios';
+import { TRow } from './types';
+
+const RowTable = ({ appointment }:TRow) => {
+  const {
+    isAvailable, isBooked, datetime, id,
+  } = appointment;
+  const [availability, setAvailability] = useState<boolean>(isAvailable);
+  return (
+    <TableRow sx={{ backgroundColor: availability ? 'white' : '#ffe766' }}>
+      <TableCell sx={bodyCell}>
+        {`${datetime.slice(5, 10)} | ${datetime.slice(12, -8)}`}

```

Comment:
we need to convert UTC date to local time, please do your research and find a way using `moemntjs` to do